source/DateLine.mc

```diff
@@ -37,6 +41,9 @@ class DateLine extends Ui.Drawable {
 		mX = params[:x];
 		mY = params[:y];
 		mYLine2 = params[:yLine2];
+		
+		// Localized letter for week number
+		mWeekOfYearLetter = "W";
```

Comment:
Need help with this :)

Reply:
Hi @wicol am I right, that you have issues understanding the mechanics on Internationalization?
I think it would be something like
`
var rezStrings = Rez.Strings;
`
and then
`
rezStrings.calendarWeek
`
See the `draw`-Function in this file.